taggit/managers.py

```diff
@@ -365,12 +356,6 @@ def similar_objects(self):
             results.append(obj)
         return results
 
-    # _TaggableManager needs to be hashable but BaseManagers in Django 1.8+ overrides
```

Comment:
Are you sure this is obsolete? Based on the comment, it seems it would still be relevant -- was it fixed in a later version of Django? Unclear to me looking at 1128c640cad63b98e29313798b1a42927c17ec17 if there's a regression test for this or not (maybe an existing test failed).

Reply:
In Django 1.8, `BaseManager` did not define its own `__hash__` method. Observe:

https://github.com/django/django/blob/1.8/django/db/models/manager.py#L53-L237

Starting with Django 1.9, it does:

https://github.com/django/django/blob/1.9/django/db/models/manager.py#L234-L235

So it is no longer necessary for classes extending `BaseManager` to define a `__hash__`.

Do you agree?

Reply:
Yes, thanks for checking.